backend-forms.md

```diff
@@ -363,6 +364,18 @@ Radio lists can also support a secondary description.
 
 Radio lists support three ways of defining the options, exactly like the [dropdown field type](#field-dropdown). For radio lists the method could return either the simple array: **key => value** or an array of arrays for providing the descriptions: **key => [label, description]**
 
+<a name="field-balloon"></a>
+### Balloon Selector
+
+`balloon-selector` - renders a list, where only one item can be selected at a time.
+
+    gender:
+        label: Gender
+        type: balloon-selector
+        options:
+            female: Female
+            male: Male
+
```

Comment:
Please add the extra information from the checkboxlist widget (excluding the bit about secondary descriptions, as `balloon-selector` doesn't actually currently support them.

```
Balloon selectors support three ways of defining the options, exactly like the [dropdown field type](#field-dropdown).
```